### ISSUE TITLE
pokestops are displayed in the wrong places when using mysql

### ISSUE THREAD

**Issue by Yossi** [AHAAAAAAA/PokemonGo-Map#1791]
## Expected Behavior
Pokestops should show their correct locations

## Current Behavior
Gyms and pokestops seem to line up in columns rather than where they belong.
Note that when using sqlite they display correctly.

## Possible Solution
Perhaps the lat & lng fields need to store more precision 

## Steps to Reproduce (for bugs)
1. Configure to use mysql
2. Wait for pokestops to be rendered

## Your Environment
* Version used: develop branch at 2c04f78b7977c1a9caf45313cef7710ee843cea3
* Environment name and version: Python 2.7.3 Ubuntu 12.04.5 MySQL server 5.5.50

**Comment by kevinpark1217** [comment]
Yes, it seems like the scan areas are off as well. The problem immediately disappears as soon as I switch back to sqlite.

+1

**Comment by kevinpark1217** [comment]
Currently MySQL is only returning up to 4 decimal places and 3 decimal places for latitude and longitude. But in order for these coordinates to be accurate, I believe that latitude and longitude have to be at least 6 and 7 decimal places accordingly.

**Comment by Yossi** [comment]
I fixed this by doing a search/replace s/FloatField/DoubleField/ in models.py and then dropping all the tables in MySQL.
